src/Auth/AuthCookie.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Authentication Cookie.
+ *
+ * @package WPGraphQL\Login\Auth
+ * @since 0.0.1
+ */
+
+ // phpcs:disable WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
Don't think we need this.

```suggestion
```

Reply:
I added this because I was using wordpress filters which are not starting with `graphql_login_`. So I added this disable lint problem.

Reply:
Updated and used the phpcs:ignore per line.